python/ql/src/Security/CWE-022/PathInjection.ql

```diff
@@ -22,5 +22,5 @@ import DataFlow::PathGraph
 
 from Configuration config, DataFlow::PathNode source, DataFlow::PathNode sink
 where config.hasFlowPath(source, sink)
-select sink.getNode(), source, sink, "$@ flows to here and is used in a path.", source.getNode(),
-  "User-provided value"
+select sink.getNode(), source, sink, "This path depends on $@.", source.getNode(),
+  "a user-provided value"
```

Comment:
I'm rather rusty on how these select statements look in results. Am I right to think that this is shown as:

> This path depends on [a user-provided value](source.getNode).

If so, this looks good to me 👍🏻

Reply:
Yep, that's right!